libexec/configure/actions/help

```diff
@@ -183,4 +184,27 @@ help_for_clocksource() {
 EOF
 }
 
+help_for_dropcache() {
+  cat <<EOF
+  SYNOPSIS:
+
+    alces configure dropcache <pagecache|slabobjs|both|1|2|3>
+
+  DESCRIPTION:
+
+    Clears the vfscache. Does not prevent the cache from filling again.
```

Comment:
Allows an administrator user to write a value to /proc/sys/vm/drop_caches causing the kernel to drop clean caches from the pagecache and/or reclaimable slab objects like dentries and inodes.  Once dropped, their memory becomes free.

Specify 'pagecache' to drop clean caches from the pagecache, 'slabobjs' to free reclaimable slab objects like dentries and inodes or 'both' to free both the pagecache and slab objects.